### PR TITLE
Button: splitbutton divider fix and SplitButton styles reorg

### LINKED ISSUE #10960: Separator in CommandBar missing for enabled actions
### Environment Information

- **Package version(s)**: starting with office-ui-fabric-react@7.36.0
- **Browser and OS versions**: any

### Please provide a reproduction of the bug in a codepen:

Check out:
https://codepen.io/LukeO/pen/XWWRNXg

And load "office-ui-fabric-react@7.36.0" to see the difference.

#### Actual behavior:

Since office-ui-fabric-react@7.36.0 there is no separator/divider for split and enabled actions in the CommandBar. There is a separator for disabled actions, though.

#### Expected behavior:

There should either always be a divider or never. But not depending on the enable/disabled setting.
Even better would be an option to choose...

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Low

### PR DESCRIPTION
**Pull request by maxwellred**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #10960
- [x] Include a change request file using `$ yarn change`

#### Description of changes

This is a fix of the aforementioned issue but is also a transfer of all SplitButton styles out of ButtonThemes.ts and into SplitButton.styles.ts. 

#### CommandBar enabled splitbutton missing divider BEFORE
Screen Shot 2019-11-06 at 1 48 26 PM

#### CommandBar enabled splitbutton missing divider AFTER
Screen Shot 2019-11-06 at 1 49 18 PM

#### High Contrast styles confirmed unaffected
spolitbutton_HS_styles

#### Custom splitbutton issue BEFORE
Screen Shot 2019-10-29 at 11 27 13 AM

#### Custom splitbutton issue AFTER
Screen Shot 2019-10-29 at 11 27 36 AM

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11099)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Button/CommandBarButton/CommandBarButton.styles.ts [hunk 1]
```diff
@@ -2,13 +2,14 @@ import { IButtonStyles } from '../Button.types';
 import { ITheme, concatStyleSets, getFocusStyle, HighContrastSelector } from '../../../Styling';
 import { memoizeFunction } from '../../../Utilities';
 import { getStyles as getBaseButtonStyles } from '../BaseButton.styles';
-import { getStyles as getSplitButtonStyles } from '../SplitButton/SplitButton.styles';
+import { getStandardSplitStyles } from '../SplitButton/SplitButton.styles';
```
Comment (khmakoto) [inline packages/office-ui-fabric-react/src/components/Button/CommandBarButton/CommandBarButton.styles.ts]: Can we call it `getStandardSplitButtonStyles` instead?

Ditto with the primary version.

**Comment by maxwellred** [comment]
Hi @dzearing I was hoping to get your eyes on this re-org of styles and see if we should hold off on making this migration since many components see a large increase in size. I'm trying to conglomerate the split styles into a single source of truth but maybe there is better place to move these styles to not cause such an increase?

**Comment by micahgodbolt** [comment]
Is it possible to fix this bug without the big code re-org?

**Comment by maxwellred** [comment]
@micahgodbolt No problem, I'll shelve the reorg.

**Comment by maxwellred** [comment]
Closing this in favor of PR #11271 which doesn't have the re-org of splitbutton styles

**Pull request closed without merging**